client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -262,7 +276,11 @@ export default {
         </v-progress-circular>
       </div>
     </div>
-    <slot name="control" />
+    <slot
+      ref="control"
+      name="control"
+      @resize="onResize"
```

Comment:
I'm pretty sure you [can't do that](https://github.com/vuejs/vue/issues/4781#issuecomment-274820814)

I actually don't see where any component in our code emits an event named 'resize'

Reply:
yup that looks like a mistake that I left in there when I was trying something else before utilizing the `ResizeObserver`